It
seems
that
this
isn't
supported
","
since
there
wouldn't
be
a
good
way
to
deal
with
overflows
in
datetime.time
.
I
know
this
isn't
an
answer
directly
","
but
maybe
someone
with
more
python
experience
than
me
can
take
this
a
little
further
.
For
more
info
","
see
this
:
http://bugs.python.org/issue3250
